### ISSUE TITLE
Can not Edit or update the multi lingual category as you are doing for multi lingual post

### ISSUE THREAD

**Issue by usman-xs4arabia-dev** [binshops/laravel-blog#67]
I can see there is an option to add multi lingual categories but while editing I'm only getting the single language category. Is there any way which I'm not getting or you guys are still working on it?

**Comment by samberrry** [comment]
@usman-xs4arabia-dev yes it does not have edit, but in database it supports multi lang. Only it should be implemented in panel. 
If you are interested PRs are welcomed ;)
